content/chapters/template-chapter/lab/README.md

```diff
@@ -1,58 +1,29 @@
-# Compute
+# Template chapter
 
-The main criterion we use to rank CPUs is their _computation power_, i.e. their ability to crunch numbers and do math.
-Numerous benchmarks exist out there and they are publicly displayed on websites such as [CPUBenchmark](https://www.cpubenchmark.net/).
+This is an instance of a lab placeholder. This will be used to showcase the different highlights of the syntax.
 
-This benchmark measures the performance of the computer's CPU in a variety of scenarios:
-- its ability to perform integer operations
-- its speed in floating point arithmetic
-- data encryption and compression
-- sorting algorithms and others
+In order to use links to to outside resources, you can add them as [such](http://example.com).
 
-You can take a look at what exactly is measured using [this link](https://www.cpubenchmark.net/cpu.php?cpu=AMD+Ryzen+Threadripper+PRO+5995WX).
-It displays the scores obrtained by a high-end CPU.
-Apart from the tests above, other benchmarks can also focus on other performance metrics such as branch prediction or prefetching.
+This is a list of all the best cuisines in the world
+- Indian
+- Italian
+- French
```

Comment:
I beg to disagree, but we'll leave it like this.

Reply:
You would disagree, and you would be wrong.